### DISCUSSION TITLE
MAUI's webview on Linux should be prioritized

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/maui#22159] [General]
Blazor.Webassembly doesn't support Task.Result, MAUI.Blazor can
Microsoft's lacks of presence and leadership in linux UI framework can be fixed by getting that Webview working on linux
We can now truly call MAUI a cross platform framework and I can give people an experience that's unified under .NET + Web across Windows / Android / IOS / Linux

**Comment by mattleibow** [comment]
Duplicate of https://github.com/dotnet/maui/discussions/339
